### PR TITLE
[13.0][MIG] sale_product_pack: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
supersedes https://github.com/OCA/product-pack/pull/24

cc @Tecnativa TT24634

### CODE REVIEW

**Comment by ernestotejeda** [comment]
@jjscarafia could you review?.

**Comment by pedrobaeza** [comment]
Please squash administrative commits together and several translations that comes together from the same person.

**Comment by pedrobaeza** [comment]
@ernestotejeda I don't see your commit with extra changes. Have you included in @jjscarafia's one? They should be split.

**Comment by ernestotejeda** [comment]
Yes, I included my changes and the same commit. But, ok, I will divide into 2 commits

**Comment by ernestotejeda** [comment]
@pedrobaeza I have squashed administrative commits and I split the v13 migration commit into 2 commits.

**Review by jjscarafia — APPROVED** [review]
Tested and working ok. +1 for code review also
thanks for the great work @ernestotejeda 
In the future it would be great a cleaner / better approach for the lines unlink, but for now this is enough for me

File: sale_product_pack/models/sale_order.py [hunk 1]
```diff
@@ -37,3 +37,19 @@ def check_pack_line_unlink(self):
...
+            to_delete_ids = [e[1] for e in vals["order_line"] if e[0] == 2]
+            subpacks_to_delete_ids = (
+                self.env["sale.order.line"]
+                .search(
+                    [("id", "child_of", to_delete_ids), ("id", "not in", to_delete_ids)]
+                )
+                .ids
+            )
+            if subpacks_to_delete_ids:
+                for cmd in vals["order_line"]:
```
Comment (pedrobaeza) [inline sale_product_pack/models/sale_order.py:52]: What happens if one of the `subpacks_to_delete_ids` is not in the command line? That one won't get removed. You need to assure that all needed `(2, x)` commands are there.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by ernestotejeda** [comment]
@pedrobaeza , done. Check that.

**Review by pedrobaeza — APPROVED** [review]

**Comment by jjscarafia** [comment]
thanks you both!

**Pull request merged**